docs/package_info/contributing.rst

```diff
@@ -37,13 +46,15 @@ anyway so we can work with you to write the necessary tests.
 Running the Test Suite
 ~~~~~~~~~~~~~~~~~~~~~~
 
+
+
 `Github Actions <https://github.com/praw-dev/praw/actions>`_ automatically runs
 all updates to known branches and pull requests. However, it's useful to be
 able to run the tests locally. The simplest way is via:
 
 .. code:: bash
 
-   python setup.py test
+   pytest
```

Comment:
This doesn't work on my system. That may be a config error on my side, but I get the traceback
```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.6/Frameworks/Python.framework/Versions/3.7/bin/pytest", line 5, in <module>
    from pytest import main
ModuleNotFoundError: No module named 'pytest'
```

Reply:
I think this occurs due to a conflict between venv and installed. Run `pip install pytest` in your venv.

Reply:
I was able to fix my configuration and get the tests to run. But does invoking the tests in this way allow you to record new cassettes?

Reply:
Yes. `python setup.py test` was just a wrapper that did this.

Reply:
Just to confirm, `pytest` is definitely the slightly easier way to run tests.